Providers/Scripts/2.4x-2.5x/Scripts/nxPackage.py

```diff
@@ -292,16 +293,16 @@ def __init__(self, Ensure, PackageManager, Name, FilePath, PackageGroup, Argumen
         self.cmds['dpkg'][
             'absent'] = 'DEBIAN_FRONTEND=noninteractive dpkg % -r '
         self.cmds['dpkg'][
-            'stat'] = "dpkg-query -W -f='${Description}|${Maintainer}|'Unknown'|${Installed-Size}|${Version}|${Status}|${Architecture}\n' "
+            'stat'] = "dpkg-query -W -f='${Description}#@#${Maintainer}#@#'Unknown'#@#${Installed-Size}#@#${Version}#@#${Status}#@#${Architecture}\n' "
```

Comment:
And repeat the same in other places (more like search and replace with {0} and then add .format at the end.